app/assets/javascripts/analytics/ecommerce.js

```diff
@@ -0,0 +1,62 @@
+(function () {
+  "use strict";
+  window.GOVUK = window.GOVUK || {};
+
+  var Ecommerce = function (config) {
+    this.init = function (element) {
+      var self = this;
+      var ecommerceRows = element.find('[data-ecommerce-row]');
+      var startPosition = parseInt(element.data('ecommerce-start-index'));
+
+      ecommerceRows.each(function(index, ecommerceRow) {
+        var $ecommerceRow = $(ecommerceRow);
+
+        var contentId = $ecommerceRow.attr('data-ecommerce-content-id'),
+          path = $ecommerceRow.attr('data-ecommerce-path');
+
+        self._sendImpression($ecommerceRow, contentId, path, index + startPosition);
+        self._trackProductClick($ecommerceRow, contentId, path, index + startPosition);
+      });
+    }
+
+    this._sendImpression = function (_row, contentId, path, position) {
```

Comment:
Are these functions (this and `trackProductClick`) used outside of this module? If not, then do they need to be publicly available?